src/app-layout/test-classes/styles.scss

```diff
@@ -42,3 +42,18 @@
 .disable-body-scroll-root {
   /* used in tests */
 }
+.drawers-desktop-triggers-container {
```

Comment:
@just-boris I'm going to leave these in for now since they aren't hurting anything.

Reply:
I am not a big fan of any hanging code (because in the end it will become unused and misleading), but if you insist...